apps/client/src/app/components/admin-market-data/admin-market-data.component.ts

```diff
@@ -241,4 +245,46 @@ export class AdminMarketDataComponent implements OnDestroy, OnInit {
           });
       });
   }
+
+  private openCreateAssetProfileDialog() {
+    this.dataService
+      .fetchAdminMarketData({})
+      .pipe(
+        combineLatestWith(this.userService.get()),
+        takeUntil(this.unsubscribeSubject)
+      )
+      .subscribe(([adminMarketData, user]) => {
+        this.user = user;
+        const dialogRef = this.dialog.open(CreateAssetProfileDialog, {
+          autoFocus: false,
+          data: <CreateAssetProfileDialogParams>{
+            existingSymbols: adminMarketData.marketData,
+            deviceType: this.deviceType,
+            locale: this.user?.settings?.locale
+          },
+          width: this.deviceType === 'mobile' ? '100vw' : '50rem'
+        });
+
+        dialogRef
+          .afterClosed()
+          .pipe(takeUntil(this.unsubscribeSubject))
+          .subscribe((data: any) => {
+            const dataSource = data?.dataSource;
+            const symbol = data?.symbol;
+
+            if (dataSource && symbol) {
+              this.adminService
+                .addAssetProfile({ dataSource, symbol })
+                .pipe(takeUntil(this.unsubscribeSubject))
+                .subscribe({
+                  next: () => {
+                    window.location.reload();
```

Comment:
Reloading page here. Let me know if you want to call the fetchAdminMarketData again instead.

Reply:
I also did it like this. I think it's okay for the moment

Reply:
Did it the right way now

Reply:
Nice!